src/allmydata/test/test_magic_folder.py

```diff
@@ -213,32 +194,46 @@ def test_persistence(self):
 
         self.client = self.g.clients[0]
         self.stats_provider = self.client.stats_provider
-        d = self.client.create_dirnode()
-        d.addCallback(self._made_upload_dir)
+        self.collective_dircap = ""
+
+        d = self.create_invite_join_magic_folder(u"Alice", self.local_dir)
         d.addCallback(self._create_magicfolder)
 
-        def create_file(val):
+        def create_test_file(result):
             d2 = defer.Deferred()
-            self.magicfolder.set_processed_callback(d2.callback)
+            self.magicfolder.set_processed_callback(d2.callback, ignore_count=0)
```

Comment:
It's not necessary to say ``ignore_count=0`` since it's the default, and doesn't make anything clearer.